src/hssm/hssm.py

```diff
@@ -230,15 +242,29 @@ def __init__(
                         + "`model_config`."
                     )
                 self.model_config = model_config
-                self.list_params = model_config["list_params"]
+                self.list_params = model_config["list_params"][:]
             else:
                 # For supported models without configs,
                 # We don't require a model_config (because list_params is known)
                 model = cast(SupportedModels, model)
                 self.model_config = {} if model_config is None else model_config
-                self.list_params = default_params[model]
+                self.list_params = default_params[model][:]
 
         self._parent = self.list_params[0]
+        if p_outlier is not None and lapse is None:
+            raise ValueError(
+                "You have specified `p_outlier`. Please also specify `lapse`."
+            )
+        if lapse is not None and p_outlier is None:
+            raise ValueError(
+                "You have specified `lapse`. Please also specify `p_outlier`."
```

Comment:
Set to warning

You have specified the `lapse` argument, to include a `lapse` distribution. The `p_outlier` is set to either 0 or `None` however. Your lapse distribution will be ignored. Was this intended?

Reply:
Sounds good!